AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.h

```diff
@@ -34,6 +29,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)removeDelegate:(id<MSAuthTokenContextDelegate>)delegate;
 
+/**
+ * Clears cached token and account id.
```

Comment:
nit: We don't add `s` to verbs in docs with third person singular:

`Clear`, `Set` and `Retrieve`.